.github/workflows/css.yml

```diff
@@ -44,3 +44,15 @@ jobs:
 
       - name: Run CSS tests
         run: npm run css-test
+
+      # Check that there are no stylelint multiline comments (e.g. /* stylelint-disable */)
+      - name: Check built CSS files
```

Comment:
Are we sure this `find` command is runnable on all platforms?

Reply:
Can't we do anything about that by adding a stylelint rule ? (https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/comment-no-loud/README.md)

Reply:
@MewenLeHo We can't put it in `npm run css-lint` because there's not necessary a `dist` dir at this moment. So we should create a specific `npm` command for that + change it because `bc` doesn't exist on Windows. IMO it's not worth it since we have the CI checking it for us. We rarely use `stylelint-disable`.

@louismaximepiton I'm not sure it can work because of the rtl multi-line comments. Otherwise, we will need to add `stylelint-disable` for each RTL comment and I'm not sure it will work.